yugabyte/src/yugabyte/core.clj

```diff
@@ -317,6 +317,15 @@
         colocated (and (not (utils/is-test-geo-partitioned? opts)) (> (rand) 0.5))]
     (assoc opts :yb-packed-columns-enabled packed-columns-enabled :yb-colocated colocated)))
 
+(defn test-3
```

Comment:
is duplicate definition of test-3 allowed ?

Reply:
some merging issue, code is duplicated above for some reason.
thanks!